rotkehlchen/globaldb/upgrades/v5_v6.py

```diff
@@ -0,0 +1,63 @@
+import logging
+from typing import TYPE_CHECKING
+
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.types import GeneralCacheType
+
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.drivers.gevent import DBConnection, DBCursor
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+V5_V6_UPGRADE_UNIQUE_CACHE_KEYS: set[GeneralCacheType] = {
+    GeneralCacheType.CURVE_POOL_ADDRESS,
+    GeneralCacheType.MAKERDAO_VAULT_ILK,
+    GeneralCacheType.CURVE_GAUGE_ADDRESS,
```

Comment:
This should also be upgraded to match the `UNIQUE_CACHE_KEYS` you have in globaldb/cache.py https://github.com/rotki/rotki/pull/5857/files#diff-1cb4a4341fba862e07859a031fda9e9cc0bdacb3ca4023307c9343c0d375da89R13

```
UNIQUE_CACHE_KEYS: set[GeneralCacheType] = {
    GeneralCacheType.CURVE_POOL_ADDRESS,
    GeneralCacheType.MAKERDAO_VAULT_ILK,
    GeneralCacheType.CURVE_GAUGE_ADDRESS,
    GeneralCacheType.YEARN_VAULTS,
}
```